data/settingsMenuItems.ts

```diff
@@ -0,0 +1,46 @@
+import {AvalancheCenterID} from 'types/nationalAvalancheCenter';
+
+const settingsMenuItems: Record<string, {title: string; url: string}[]> = {
+  NWAC: [
+    {
+      title: 'About NWAC',
+      url: 'http://www.nwac.us/about/about-us',
+    },
+    {
+      title: 'Avalanche Education (Learn)',
+      url: 'http://www.nwac.us/about/about-us',
+    },
+    {
+      title: 'Become a Member',
+      url: 'http://www.nwac.us/about/about-us',
+    },
+    {
+      title: 'Volunteer with NWAC',
+      url: 'http://www.nwac.us/about/about-us',
+    },
+    {
+      title: 'Support NWAC',
+      url: 'http://www.nwac.us/about/about-us',
+    },
+    {
+      title: 'Latest News',
+      url: 'http://www.nwac.us/about/about-us',
+    },
+  ],
+  SNFAC: [
+    {
+      title: 'About SNFAC',
+      url: 'http://www.sawtoothavalanche.com/about/',
+    },
+    {
+      title: 'Donate',
+      url: 'https://friends.sawtoothavalanche.com/make-a-donation/',
+    },
+    {
+      title: 'News and Events',
+      url: 'https://friends.sawtoothavalanche.com/news-events/',
+    },
+  ],
+};
+
+export const getMenuItemsForCenter = (avalancheCenterId: AvalancheCenterID) => settingsMenuItems[avalancheCenterId] || ([] as {title: string; url: string}[]);
```

Comment:
Isn't this also an invalid TypeScript cast? The array items do *not* have title and url. I think the correct type is 

```ts
export const getMenuItemsForCenter = (avalancheCenterId: AvalancheCenterID): {title: string, url: string}[] | undefined => settingsMenuItems[avalancheCenterId]
```
```

Reply:
I intentionally didn’t want to return undefined, so this cast means that the empty array is narrowed to the same type as the non-empty one. Not sure what is invalid here?

Reply:
`[]` does not have `title` or `url` properties in its members - yes, you'd never be able to tell since it has no items in it, but it's not a correct type. 

Also it looks like without any type gymnastics, the inferred type here is correct:

![Screenshot from 2023-09-08 07-41-59](https://github.com/stevekuznetsov/avalanche-forecast/assets/7328370/cd2c7bd0-3a8f-4cb3-9d25-d80ae981cd56)

Just seems like if we're going to use TypeScript, using the escape hatch should be something we do sparingly and carefully.

Reply:
I don't know that I did a good job communicating it -

> Not sure what is invalid here?

Using the literal ` as ` to override inferred types instead of changing the program to be correct as per `tsc`.